i18n/locales/source/dashboard/unplugged.yml

```diff
@@ -188,3 +188,6 @@ en:
       courseF_unplugged_powerOfWords:
         title: Unplugged Activity
         desc: Today you will learn about what to do if someone is being mean online.
+      deeper learning description:
+        title: ''
+        desc: 'translation missing: en-US.data.unplugged.desc'
```

Comment:
Nah. This looks like a thing that was created by accident and is probably not used. Let's just leave it, hide in Crowdin, and eventually we'll clean up stuff like this. I'm having Fikayo work on i18n stuff so we may actually get to doing clean up this summer!